src/Result.php

```diff
@@ -16,13 +18,17 @@ public function isValid(): bool
         return $this->errors === [];
     }
 
-    public function addError(string $message): void
+    public function addError(string $message, $key = null): void
```

Comment:
The method now needs phpdoc since it's not clear what `$key` is from method signature.

Reply:
Added. See https://github.com/yiisoft/validator/pull/151#discussion_r792535726.

Reply:
:heavy_check_mark:

Reply:
I've meant description of what `$key` is for, not just types.

Reply:
OK. Added.